.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**WARNING: Please only use GitHub issues for BUG REPORTS and FEATURE REQUESTS**
+Lengthy algorithm discussions etc are fine too, but also consider creating a wiki page or a page on website.
+
+Here is what to do with other types of questions:
+1. Build problems: ask in #help channel in our discord: https://discord.gg/pKujYxD
+2. Configuration questions: ask in #help channel in our discord: https://discord.gg/pKujYxD
+3. Running problems: ask in #help channel in our discord: https://discord.gg/pKujYxD
+
+
+If you are filling a bug report, please fill the fields below.
```

Comment:
*filing*

Reply:
Hmmm I meant `fill` (as in "fill the form"), not `file`..
Although I guess `filing` is what people do with bug reports?..

Reply:
I thought you meant "filing" and it was a typo. Anyway, I'm not a native speaker.

Reply:
You usually file a bug report, so filing is correct.

Reply:
I don't like 'filing' here followed with 'fill' a few words later. Alternative suggestion: 'If you are submitting a bug report, please complete the fields below.'